web/cobrands/bathnes/assets.js

```diff
@@ -6,7 +6,7 @@ if (!fixmystreet.maps) {
 
 fixmystreet.maps.banes_defaults = {
     http_options: {
-        url: "https://isharemaps.bathnes.gov.uk/getows.ashx",
+        url: "https://tilma.mysociety.org/proxy/bathnes/isharemaps/",
```

Comment:
Do we want the live service to use the proxy, presumably not?

Reply:
Have changed this so staging sites use tilma and live sites use their one.